enterprise/causal-clustering/src/main/java/org/neo4j/causalclustering/core/consensus/outcome/Outcome.java

```diff
@@ -72,7 +74,7 @@
     private FollowerStates<MemberId> followerStates;
     private Collection<ShipCommand> shipCommands = new ArrayList<>();
     private boolean electedLeader;
-    private boolean steppingDown;
+    private Optional<Long> steppingDownInTerm;
```

Comment:
`OptionalLong` - long, int and boolean have specialised versions of Optional and Stream.

Reply:
Nice - did not know. I'll sort